src/Logging/JUnit/TestCase.php

```diff
@@ -38,27 +38,39 @@ class TestCase
     public $assertions;
 
     /**
-     * @var string
+     * @var string|float (a stringified float, from phpunit XML output)
      */
     public $time;
 
     /**
-     * Number of failures in this test case
+     * List of failures in this test case
      *
      * @var array
      */
     public $failures = [];
 
     /**
-     * Number of errors in this test case
-     * TODO: Not a number?
+     * List of errors in this test case
      *
      * @var array
      */
     public $errors = [];
 
+    /**
+     * List of errors in this test case
```

Comment:
~~errors~~**skipped**

actually, these comments are amazingly obvious aren't they?
```
/** @var array */
public $skipped = [];
```
should do the job and even that annotation should be useless because the type is known by declaration